mindsdb/integrations/handlers/llama_index_handler/llama_index_handler.py

```diff
@@ -304,3 +282,37 @@ def _setup_index(self, documents):
         )
 
         return index
+
+    def _get_llama_index_api_key(self, args, strict=True):
```

Comment:
There is a util function for this so better to use one in all handlers https://github.com/mindsdb/mindsdb/blob/82dfb9750a3bd3dfec5adc86db5ba6c64a204fd2/mindsdb/integrations/utilities/handler_utils.py#L10

Reply:
@ZoranPandovski I implemented this in the previous PR please check, this util can only be used to take input of `api_key`, what I want is `openai_api_key` and `github_token`. So I had to revert my changes #8593

Reply:
For reference: relevant PR that should fix what you stumbled upon is #8245 (@martyna-mindsdb), relevant issue is #7496.

Reply:
Why don't we just add one more parameter to the `get_api_key()` such as key which will be passed accordingly @paxcema

Reply:
[reverted change](https://github.com/pranavvp16/mindsdb/blob/4fd457cbc5c4d3b4af54339b2825114c3b9c340f/mindsdb/integrations/utilities/handler_utils.py) something like this, which will make the utils usable to great extent. Also I searched for usage, the util has somewhat 5 usages, So we can change them too. If this isn't possible please suggest me a change, as this makes the llama_index handler unusable totally. And I have built my hackathon project using it!

Reply:
@paxcema #8245 adds a prefix `{name}_api_key` which won't work for `github_token`. Please look at the [reverted changes](https://github.com/pranavvp16/mindsdb/blob/4fd457cbc5c4d3b4af54339b2825114c3b9c340f/mindsdb/integrations/utilities/handler_utils.py) this seems the best approach to me.

Reply:
Let's go for this as a hotfix for now, because the handler is broken atm @pranavvp16 @ZoranPandovski. But we need to refactor this ASAP as there's a lot of duplicated logic going on at the moment

Reply:
@pranavvp16 please put a `# TODO` comment on this so that we can refactor ASAP